src/app/users/users-routing.module.ts

```diff
@@ -37,6 +39,19 @@ const routes: Routes = [
           resolve: {
             usersTemplate: UsersTemplateResolver
           }
+        },
+        {
+          path: ':id',
+          children: [
+            {
+              path: '',
+              component: ViewUserComponent,
+              data: { title: extract('View User'), routeResolveBreadcrumb: ['user', 'username'] },
```

Comment:
```suggestion
```
You just had to move the resolver not this.

Reply:
@abhaychawla If I leave it there the route doesn't show the username, I had to move it next to the resolver.